tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -1015,7 +1020,247 @@ def test_multiple_triggers_threshold_period(self):
             [other_action, self.action],
             [
                 (rule.resolve_threshold - 1, IncidentStatus.CLOSED),
-                (rule.resolve_threshold - 1, IncidentStatus.WARNING),
+                (rule.resolve_threshold - 1, IncidentStatus.CLOSED),
+            ],
+        )
+
+    def setup_for_multiple_triggers_multiple_actions_test(self):
```

Comment:
Most of the changes here are just to make the old tests work with the new behavior. The only exceptions being the following four new tests which each checks one of the metric alert permutations outlined in the Notion doc tables.

Reply:
Thanks for adding these tests! Could you add a docstring/top-level comment to each test explaining which scenario the test is for? The test titles can be a bit harder to parse

Reply:
Sure sounds good :)